packages/components/inputs/input-utils/src/styles.ts

```diff
@@ -93,7 +93,7 @@ const getInputStyles = (props: TInputProps) => {
       : 'unset'}; /* fix for mobile safari */
     outline: none;
     overflow: hidden;
-    padding: ${designTokens.paddingForInput};
+    padding: 0 var(--spacing-30);
```

Comment:
```suggestion
    padding: 0 ${designTokens.spacing30};
```

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2785/commits/098974e6596998284f5d665ce8a0a416310509e0
Thank you